book/02-git-basics/sections/recording-changes.asc

```diff
@@ -247,7 +247,7 @@ doc/*.txt
 doc/**/*.txt
 ----
 
-[TIP]
```

Comment:
D'après ce que je comprends du lien, les _admonitions_ c'est pour insérer des icônes et c'est utilisé de la manière suivante :
TIP: blablabla
"The label must be uppercase and followed by a colon (:)."
Ce qui ne me semble pas correspondre avec l'utilisation faite ici.